docs/en/observability/cloud-monitoring/aws/monitor-aws-waf-firehose.asciidoc

```diff
@@ -0,0 +1,141 @@
+[[monitor-aws-waf-firehose]]
+= Monitor Web Application Firewall (WAF) logs
+
+++++
+<titleabbrev>Monitor WAF logs</titleabbrev>
+++++
+
+In this section, you'll learn how to send AWS WAF events from AWS to your Elastic stack using Amazon Data Firehose.
+
+You will go through the following steps:
+
+- Select a WAF-compatible resource (CloudFront distribution)
+- Create a delivery stream in Amazon Data Firehose
+- Create a web Access Control List (ACL) to generate WAF logs
+- Set up logging to forward the logs to the Elastic stack using a Firehose stream
+- Visualize your WAF logs in {kib}
+
+[discrete]
+[[firehose-cloudtrail-prerequisites]]
+== Before you begin
+
+We assume that you already have:
+
+- An AWS account with permissions to pull the necessary data from AWS.
+- A deployment using our hosted {ess} on {ess-trial}[{ecloud}]. The deployment includes an {es} cluster for storing and searching your data, and {kib} for visualizing and managing your data. AWS Data Firehose works with Elastic Stack version 7.17 or greater, running on Elastic Cloud only.
+
+IMPORTANT: Make sure the deployment is on AWS, because the Amazon Data Firehose delivery stream connects specifically to an endpoint that needs to be on AWS.
+
+[discrete]
+[[firehose-waf-step-one]]
+== Step 1: Install AWS integration in {kib}
+
+. In {kib}, navigate to *Management* > *Integrations* and browse the catalog to find the Amazon Data Firehose integration.
+
+. Navigate to the *Settings* tab and click *Install Amazon Data Firehose assets*.
```

Comment:
The instructions in this guide require the AWS integration (we are using the `logs-aws.waf-default` data stream)

Since the Amazon Data Firehose integration setup also requires AWS integration, which one should I mention here?

![CleanShot 2024-05-07 at 16 21 44](https://github.com/elastic/observability-docs/assets/25941/115672ad-e876-4630-895e-eac683d9f7ee)

Reply:
@zmoog  sure I get the point.... let's review this on Monday